stories/src/table/with-dimension-locking.jsx

```diff
@@ -191,10 +191,7 @@ export default class TableApp extends Component<AppProps, AppState> {
     });
 
     // dropped outside the list
-    if (
-      !result.destination ||
-      result.destination.index === result.source.index
```

Comment:
It is possible to move an item back into the same spot. The check ` result.destination.index === result.source.index` will exit early if that is the case

Reply:
If you check the lines just below it, the same check occurs, making it redundant. https://github.com/atlassian/react-beautiful-dnd/blob/8e4c3845e8d396e8baf5d95c6077d49f09b9913b/stories/src/table/with-dimension-locking.jsx#L199

Reply:
Oh i see now!